components/brave_rewards/resources/ui/components/modalRedirect/index.tsx

```diff
@@ -77,13 +79,19 @@ export default class ModalRedirect extends React.PureComponent<Props, {}> {
                 : errorText
               }
               </p>
+              {
+                learnMore
```

Comment:
A common React idiom is to use a logical expression for optionally-rendered things:

```jsx
learnMore &&
  <StyledLink ...>...</StyledLink>
```